scheduler/src/cook/mesos/util.clj

```diff
@@ -98,8 +98,8 @@
          [job-ent & job-ents] job-ents]
     (if job-ent
       (let [{:keys [cpus mem]} (job-ent->resources job-ent)]
-        (recur (+ total-cpus cpus)
-               (+ total-mem mem)
+        (recur (+ total-cpus (or cpus 0))
```

Comment:
No, but incomplete jobs are used in tests.   At a practical level, we either have to change the tests to pick arbitrary resource amounts, or change this function so that it doesn't throw a NullPointerException if any jobs don't have explicit cpu or mem requirements.